tests/test_binary_sensors.py

```diff
@@ -10,36 +10,64 @@
 
 
 class TestAlertSensor:
+    def test_binary_sensor_is_on(self, hass, config_entry, alarm_device):
+        alarm_device.connect("username", "password")
+        alarm_device.update()
```

Comment:
In another PR, put this initialization in the `conftest.py` under `alarm_device` fixture: https://github.com/palazzem/ha-econnect-alarm/blob/71c1d36/tests/conftest.py#L23-L37

Then you can remove all these lines and fix any other test that is not passing (some tests may assume the device is always initialized, in which case you should fix it to uninitialize it again).